foundry.toml

```diff
@@ -1,13 +1,58 @@
+# Full reference https://github.com/foundry-rs/foundry/tree/master/config
+
 [profile.default]
-src = "src"
-out = "out"
-libs = ["lib"]
-verbosity = 3
-solc_version = '0.8.20'
+  auto_detect_solc = false
+  block_number = 19537700 # Mar-29-2024 05:52:47 AM +UTC
+  bytecode_hash = "none"
+  cbor_metadata = false
+  fuzz = { runs = 1_000 }
+  gas_reports = ["*"]
+  libs = ["lib"]
+  optimizer = true
+  optimizer_runs = 10_000
+  out = "out"
+  script = "script"
+  solc = "0.8.20"
+  src = "src"
+  test = "test"
+  evm_version = 'london'
+  verbosity = 2
+
+[profile.ci]
+  fuzz = { runs = 10_000 }
+  verbosity = 4
 
-# Testing
-block_number = 19537700 # Mar-29-2024 05:52:47 AM +UTC
+[etherscan]
+  arbitrum_one = { key = "${API_KEY_ARBISCAN}", chain = 42161 }
+  # avalanche = { key = "${API_KEY_SNOWTRACE}", chain = 43114 }
+  # bnb_smart_chain = { key = "${ API_KEY_BSCSCAN}", chain = 56 }
+  gnosis_chain = { key = "${API_KEY_GNOSISSCAN}", chain = 100 }
+  # goerli = { key = "${API_KEY_ETHERSCAN}", chain = 5 }
+  mainnet = { key = "${API_KEY_ETHERSCAN}", chain = 1 }
+  optimism = { key = "${API_KEY_OPTIMISTIC_ETHERSCAN}", chain = 10 }
+  polygon = { key = "${API_KEY_POLYGONSCAN}", chain = 137 }
+  # sepolia = { key = "${API_KEY_ETHERSCAN}", chain = 58008 }
+
+[fmt]
```

Comment:
unless you have an explicit preference for some of the `fmt` settings, I'd keep everything default